pulsar-client/src/main/java/org/apache/pulsar/client/impl/PartitionedProducerImpl.java

```diff
@@ -342,9 +342,7 @@ public void run(Timeout timeout) throws Exception {
                 return;
             }
 
-            if (log.isDebugEnabled()) {
-                log.debug("[{}]  run partitionsAutoUpdateTimerTask for partitioned producer: {}", topic);
-            }
+            log.debug("[{}] run partitionsAutoUpdateTimerTask for partitioned producer", topic);
```

Comment:
Please don't remove `if (log.isDebugEnabled())`. We use `if (log.isDebugEnabled())` for debug level logging everywhere in the project.

Reply:
Thank you @sijie . What's the benefits of using log.isDebugEnabled here?  From what I understood, ```log.isDebugEnabled``` only useful for performance consideration, and the the method ```log.debug``` will also check if current log level is DEBUG.

Reply:
avoid executing the code in if branch (e.g. avoid string conversation). in this line you are fixing, it is probably okay. however for keeping consistent coding convention, it would be good to stick to the original style.

Reply:
OK. I reverted this change.